services/QuillLMS/app/models/segment_integration/user.rb

```diff
@@ -73,6 +78,16 @@ def school_params
       end
     end
 
+    def onboarding_group_params
+      return {} if admin? || !teacher?
```

Comment:
Are there other roles than teacher that we're looking for?

Otherwise, maybe: `return {} unless teacher?`

Reply:
@brendanshean the problem here is that technically, all admins are teachers!

Reply:
That's right.  My mistake.